frontend/src/components/Item/Metadata/ImageSearch.vue

```diff
@@ -3,32 +3,36 @@
     :model-value="dialog"
     :fullscreen="$vuetify.display.mobile"
     content-class="image-search-dialog-content"
-    width="60%"
-    @click:outside="$emit('update:dialog', false)">
+    @update:model-value="(value) => emit('update:dialog', value)">
     <v-card height="100%" class="image-search-card">
-      <v-card-title>{{ $t('search.name') }}</v-card-title>
+      <v-card-title>{{ t('search.name') }}</v-card-title>
       <v-divider />
       <v-row align="center" class="mx-16 my-4">
         <v-select
           v-model="source"
           class="mx-4"
           :items="sources"
           :disabled="loading"
-          :label="$t('metadata.source')"
+          :label="t('metadata.source')"
+          :placeholder="t('metadata.sourceAll')"
+          persistent-placeholder
           variant="outlined"
-          hide-details />
+          hide-details
+          clearable />
```

Comment:
This sets the value to `null` and makes the next queries fetch a `null` provider, instead of removing the parameter. I think there should a check for null in the query below :)